src/Stratis.Bitcoin/P2P/IPeerAddressManager.cs

```diff
@@ -32,7 +32,13 @@ public interface IPeerAddressManager : IDisposable
         /// </para>
         /// </summary>
         void AddPeers(IPEndPoint[] endPoints, IPAddress source);
-
+        
+        /// <summary>
+        /// Remove a peer from the <see cref="Peers"/> dictionary.
+        /// <para>
+        /// Only routable IP addresses will be removed since only routable IP addressed can be added. <see cref="IpExtensions.IsRoutable(IPAddress, bool)"/>
```

Comment:
use `<seealso>` instead of `<see>` and put it OUTSIDE of summary section

Reply:
Looking at the comments on the 2 methods above, this looks in line with the current way of doing things...
Hopefully not a blocker...

Reply:
correctly used on line 23, incorrectly on 31 and here